src/components/DocumentControlsPanel/DocumentActionsPanel/DocumentActionsPanel.cypress.js

```diff
@@ -71,24 +71,24 @@ describe("DocumentActions", function() {
     cy.visit(`?collection=${this.collectionId}`);
 
     // Verify that the correct document is selected in the list
-    cy.get("[data-testid=document_action_document_name]").should(
+    cy.get("[aria-label='Document name in document actions panel']").should(
```

Comment:
Get the doc using `getByText` based on the doc's name that should be displaying in the UI.